admin_guide/high_availability.adoc

```diff
@@ -292,28 +292,6 @@ done;
 ----
 ====
 
-. Depending on your environment policies, you can either reuse the *router*
-service account created previously or create a new *ipfailover* service account.
-+
-Ensure that either the *router* service account exists as described in
-xref:../install_config/router/index.adoc#install-config-router-overview[Deploying a Router] or create
-a new *ipfailover* service account. The example below creates a new service
-account with the name *ipfailover* in the *default* namespace:
-+
-====
-----
-$ oc create serviceaccount ipfailover -n default
-----
-====
-
-. Add the *ipfailover* service account in the *default* namespace to the *privileged* SCC:
-+
-====
-----
-$ oadm policy add-scc-to-user privileged system:serviceaccount:default:ipfailover
```

Comment:
I think this step should NOT be deleted.  `oc adm ipfailover` can help create the ipfailover service account. but still need this step to add to scc.